src/input_cp2k_global.F

```diff
@@ -723,9 +723,9 @@ SUBROUTINE create_fm_diag_rules_section(section)
 
       CALL keyword_create(keyword, __LOCATION__, name="ELPA_FORCE_REDISTRIBUTE", &
                           description="Controls how to perform redistribution when ELPA is used for diagonalization. "// &
-                          "By default, matrices are redistributed only to prevent crashes in the ELPA library "// &
-                          "which happens when the original matrix is distributed over too many processors. "// &
-                          "By turning on this keyword, redistribution is always performed using the defined rules. ", &
+                          "By default, redistribution is alway performed using the defined rules. "// &
```

Comment:
typo: alway -> always

Reply:
Thanks. My editor doesn't like CP2K's formatting and split the line into multiple ones. I ended up removing an extra character by mistake when trying to manually fix things.

Reply:
Out of curiosity, what does your editor use for formatting Fortran code?

Reply:
It uses [fortls](https://github.com/fortran-lang/fortls), but I never really configured it properly (yet). I need to find some time to configure it properly. ;)